sentry/resource_sentry_project.go

```diff
@@ -83,6 +83,10 @@ func resourceSentryProject() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"resolve_age": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
```

Comment:
Can we add Optional: true and a description?

```suggestion
			"resolve_age": {
				Type:        schema.TypeInt,
				Optional:    true
				Description: "Hours in which an issue is automatically resolve if not seen after this amount of time.",
				Computed:    true,
			},
```